adapools.json

```diff
@@ -882,7 +882,12 @@
         "pool_id": "32f6df36b4625d51971dedb0cabd20a6c8577165d5f15530594dbc6b",
         "member_since": "2021-05-21",
         "name": "SOLID"
+      },
+      "174": {
+        "pool_id": "2595f0a5ff4145f3d6ee09e81c2488bdfe2079004eda635a6dce1515",
+        "member_since": "2021-05-25",
+        "name": "PIGGY"
       }
```

Comment:
Please do not make changed to the adapools.json file. this file is automatically generated.